cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/dataset2/lib/kv/HBaseKVTableTest.java

```diff
@@ -44,17 +45,19 @@
   @ClassRule
   public static final HBaseTestBase TEST_HBASE = new HBaseTestFactory().get();
 
-  private static HBaseTableUtil hBaseTableUtil = new HBaseTableUtilFactory(CConfiguration.create()).get();
+  private static HBaseTableUtil hBaseTableUtil = new HBaseTableUtilFactory(CConfiguration.create(),
+                                                                           new SimpleNamespaceQueryAdmin()).get();
 
   @BeforeClass
   public static void beforeClass() throws Exception {
-    hBaseTableUtil.createNamespaceIfNotExists(TEST_HBASE.getHBaseAdmin(), NAMESPACE_ID);
+    hBaseTableUtil.createNamespaceIfNotExists(TEST_HBASE.getHBaseAdmin(),
+                                              hBaseTableUtil.getHBaseNamespace(NAMESPACE_ID));
```

Comment:
perhaps. I mean I don't know how long this operation takes. do you notice any difference? Even just code wise, seems cleaner?